pkg/webhooks/server.go

```diff
@@ -158,12 +254,34 @@ func protect(inner handlers.AdmissionHandler) handlers.AdmissionHandler {
 	}
 }
 
+func logAdmission(inner handlers.AdmissionHandler) handlers.AdmissionHandler {
+	return func(logger logr.Logger, request *admissionv1.AdmissionRequest, startTime time.Time) *admissionv1.AdmissionResponse {
+		resp := inner(logger, request, startTime)
+		if dumpPayload {
+			if dumpOperation(string(request.Operation)) && dumpKind(request.Kind.Kind) {
+				reqPayload, err := newAdmissionRequestPayload(request)
+				if err != nil {
+					logger.Error(err, "Failed to extract resources")
+					return admissionutils.ResponseFailure(err.Error())
+				}
+				if strings.EqualFold(request.Kind.Kind, "Secret") {
```

Comment:
Why the if here ?

Reply:
Do you mean like combining `if dumpPayload && if dumpOperation&& dumpKind` ?